src/modules/dashboard/components/ActionsPage/ActionsComponents/DefaultMotion.tsx

```diff
@@ -398,8 +399,117 @@ const DefaultMotion = ({
     )} | Motion | Colony - ${colony.displayName ?? colony.colonyName ?? ``}`,
   );
 
-  // @TODO will be used in the following PR
-  // const isDecision = actionType === ColonyMotions.CreateDecisionMotion;
+  const isDecision = actionType === ColonyMotions.CreateDecisionMotion;
+  const hasBanner = useMemo(
+    () =>
+      isDecision ? false : shouldDisplayMotion(currentStake, requiredStake),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [],
```

Comment:
isDecision needs to be added to the dependency array and the eslint-disable removed.